src/ray/core_worker/test/task_manager_test.cc

```diff
@@ -21,11 +21,6 @@ TaskSpecification CreateTaskHelper(uint64_t num_returns,
 }
 
 class MockActorManager : public ActorManagerInterface {
-  void PublishCreatedActor(const TaskSpecification &actor_creation_task,
-                           const rpc::Address &address) override {
-    num_publishes += 1;
```

Comment:
FYI @zhijunfu @ericl this line introduced a compilation error in Travis because now `num_publishes` is an unused variable. Fixed in https://github.com/ray-project/ray/pull/6625.

Reply:
@robertnishihara Sorry for that. and thanks for fixing it.